### DISCUSSION TITLE
Backwards compatibility with LTE UE / SIM cards

### DISCUSSION THREAD

**Discussion by o2blom** [srsran/srsRAN_Project#646] [General Help] [answered]
Hello there !

Newbie questions here. Does the new srsRAN Project support 4G/LTE user equipment ? Also, when using 5G UE, is a special 5G SIM card needed or does the 4G/LTE SIM work as well ? Blank 5G SIM cards are substantially more expensive for some reason. 

Thanks, 

/Otto

**Comment by ffxxcad** [comment, marked as answer]
I have had success with [Gialer LTE sim cards](https://www.amazon.com/stores/GIALER/LTESIMCard/page/B03F304D-60A4-49DB-B619-AD647D49DD71) on a variety of android phones but the programming is pretty limited. 

If you want to use an iPhone you will need to use 5G sim cards like [this one](https://sysmocom.de/products/sim/sysmoisim-sja5/index.html) so that you can configure SUPI concealment as described [here](https://github.com/herlesupreeth/docker_open5gs/discussions/248#discussion-5780097).

**Reply by o2blom** [reply]
Thanks for the help, I appreciate it!
